sixtyfps_runtime/corelib/layout.rs

```diff
@@ -549,7 +549,7 @@ pub fn solve_box_layout(data: &BoxLayoutData, repeater_indexes: Slice<u32>) -> S
             current_offset += 1;
             break current_offset - 1;
         };
-        res[o * 2 + 0] = layout.pos;
```

Comment:
That's not a "useless addition", this was to keep it aligned with the next line.
But that's true that not so important to keep aligned. (It used to be more important before when there was 4 entries)

Reply:
That's what the commit message also says. Shall I drop it?